recipe/meta.yaml

```diff
@@ -16,13 +16,13 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools >=46.4.0
     - wheel
```

Comment:
We probably should update the requirement for `setuptools` to reflect the upstream:
`setuptools` >=`60.2.0`
https://github.com/jupyter/jupyter-packaging/blob/main/setup.cfg#L32

Reply:
No, your pinning is from main branch but version **0.11.1** has `setuptools >=46.4.0` https://github.com/jupyter/jupyter-packaging/blob/6f0032414a7b3fa75070bef2fe6b95eb76053d9f/setup.cfg#L32